src/transformers/models/longformer/modeling_longformer.py

```diff
@@ -1246,6 +1246,7 @@ def forward(
         hidden_states,
         attention_mask=None,
         head_mask=None,
+        padding_len=0,
```

Comment:
I hope adding this new argument is fine.

`TFLongformerEncoder` has already this argument.

(and moving the unpadding logic here makes the code a bit easier),  i.e no need to worry `encoder_outputs[1]`,  `encoder_outputs[2]` etc in `LongformerModel`

```
        if not return_dict:
            return (sequence_output, pooled_output) + encoder_outputs[1:]
```